core/blockchain.go

```diff
@@ -653,7 +652,7 @@ func (bc *BlockChain) FastSyncCommitHead(hash common.Hash) error {
 	if block == nil {
 		return fmt.Errorf("non existent block [%x..]", hash[:4])
 	}
-	if _, err := trie.NewSecure(block.Root(), bc.stateCache.TrieDB()); err != nil {
+	if _, err := zktrie.NewSecure(block.Root(), bc.stateCache.TrieDB()); err != nil {
```

Comment:
So we totally drop any options for non-zktrie db within this PR?

Reply:
Yes, I understand that there is no reason for us to use the original trie for related operations involving state? However, in some places where block, transaction and receipt trie are calculated, the original trie is still used